backend/src/test/scala/com/softwaremill/bootzooka/user/EncryptPasswordBenchmark.scala

```diff
@@ -1,14 +1,21 @@
 package com.softwaremill.bootzooka.user
 
+import com.softwaremill.bootzooka.common.crypto.{Argon2dPasswordHashing, CryptoConfig, PasswordHashing, Salt}
 import com.softwaremill.bootzooka.common.Utils
-import com.softwaremill.bootzooka.user.domain.User
+import com.typesafe.config.Config
 
-// Run this locally to determine the desired number of iterations in PBKDF2
 object EncryptPasswordBenchmark extends App {
+  val hashing:PasswordHashing = new Argon2dPasswordHashing(new CryptoConfig {
```

Comment:
can't it just use the default config here?

Reply:
done